modules/globebrowsing/globebrowsingmodule.cpp

```diff
@@ -36,19 +38,69 @@
 #include <modules/globebrowsing/tile/tileprovider/tileproviderbylevel.h>
 #include <modules/globebrowsing/tile/tileprovider/tileproviderbyindex.h>
 
+#include <openspace/engine/openspaceengine.h>
 #include <openspace/rendering/renderable.h>
 #include <openspace/util/factorymanager.h>
 
 #include <ghoul/misc/templatefactory.h>
 #include <ghoul/misc/assert.h>
 
+#include <ghoul/systemcapabilities/generalcapabilitiescomponent.h>
+
 namespace openspace {
 
-GlobeBrowsingModule::GlobeBrowsingModule() : OpenSpaceModule("GlobeBrowsing") {}
+GlobeBrowsingModule::GlobeBrowsingModule()
+: OpenSpaceModule("GlobeBrowsing")
```

Comment:
in modules/globebrowsing/cache/memoryawaretilecache.cpp you said it should be on a new line?